erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -315,7 +363,11 @@ int ExternalOutput::deliverAudioData_(char* buf, int len) {
 int ExternalOutput::deliverVideoData_(char* buf, int len) {
     if (videoSourceSsrc_ == 0){
       RtpHeader* h = reinterpret_cast<RtpHeader*>(buf);
-      videoSourceSsrc_ = h->getSSRC();
+      if (h->getPayloadType() == RED_90000_PT) {
```

Comment:
this is a precious information you gave me. Thanks a lot. Tomorrow morning i'll test without fec